src/olympia/promoted/templates/promoted/emails/stripe_charge_succeeded.txt

```diff
@@ -0,0 +1,7 @@
+{% block content %}
+Hello,
+
+We received a notification from Stripe about a successful payment, see: {{ stripe_payment_url }}
+
```

Comment:
I would have expected more info in there, but I guess it's good enough for now.

Reply:
If you have some ideas, let me know.

Reply:
We'll see what admins would like to receive when they start getting some emails :)